frontend/src/components/ProjectSelect.vue

```diff
@@ -113,7 +113,7 @@ export default defineComponent({
 
     watchEffect(prepareProjectList);
 
-    const hasAdminFeature = computed(() =>
+    const hasRbacFeature = computed(() =>
```

Comment:
My bad, could you please change  hasRbacFeature to hasRBACFeature? I find existing code use the RBAC convention.

Reply:
NP, updated